test/e2e/pd.go

```diff
@@ -50,10 +51,15 @@ var _ = Describe("PD", func() {
 
 		nodes, err := c.Nodes().List(labels.Everything(), fields.Everything())
 		expectNoError(err, "Failed to list nodes for e2e cluster.")
-		Expect(len(nodes.Items) >= 2).To(BeTrue())
 
-		host0Name = nodes.Items[0].ObjectMeta.Name
-		host1Name = nodes.Items[1].ObjectMeta.Name
+		numHosts = len(nodes.Items)
```

Comment:
FYI @quinton-hoole @ixdy - here's on e2e test that assumes a cluster has at least two nodes, and then explicitly uses the first two nodes in the cluster for the test. If we want e2es to run on larger clusters (or in parallel), we need to fix this.